recipes/bioconductor-txdb.hsapiens.ucsc.hg38.knowngene/post-link.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+FN="TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0.tar.gz"
+URLS=(
+    "https://depot.galaxyproject.org/software/TxDb.Hsapiens.UCSC.hg38.knownGene/TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0_src_all.tar.gz"
+    "http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0.tar.gz"
+    )
+MD5="1d5e07631ea58e96b11905d39e76ca6e"
+
+# Use a staging area in the conda dir rather than temp dirs, both to avoid
+# permission issues as well as to have things downloaded in a predictable
+# manner.
+STAGING=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG-BUILDNUM
```

Comment:
We could offer an ENV variable that people can use to overwrite this location. If available store there ... otherwise use this one here. E.g. $BIOCONDA_DATA_STAGING